build.sbt

```diff
@@ -39,16 +39,15 @@ lazy val scalatraSettings = Seq(
 
 lazy val scalatraProject = Project(
   id = "scalatra-project",
-  base = file("."),
-  settings = scalatraSettings ++ Seq(
-    name := "scalatra-unidoc",
+  base = file(".")).settings(
+    scalatraSettings ++ Seq(
+    name := "scalatra-project",
```

Comment:
Is there any reasons change the name?

Reply:
In sbt 1.0.x and later, the project name is output before the command prompt, but because `scalatra-unidoc` is displayed as the default project, it is hard to understand.

Reply:
This is artifactId for unidoc. try `sbt publishLocal`

```

[info] Packaging /Users/kenji/scalatra/target/scala-2.12/scalatra-unidoc_2.12-2.6.0-SNAPSHOT-javadoc.jar ...
[info] Done packaging.
[info] 	published scalatra-unidoc_2.12 to /Users/kenji/.ivy2/local/org.scalatra-unidoc_2.12/2.6.0-SNAPSHOT/docs/scalatra-unidoc_2.12-javadoc.jar
[info] 	published scalatra-unidoc_2.12 to /Users/kenji/.ivy2/local/org.scalatra-unidoc_2.12/2.6.0-SNAPSHOT/poms/scalatra-unidoc_2.12.pom
[info] 	published ivy to /Users/kenji/.ivy2/local/org.scalatra-unidoc_2.12/2.6.0-SNAPSHOT/ivys/ivy.xml
```

```

```

Reply:
You can customize sbt prompt 

- https://github.com/sbt/sbt/blob/v1.0.2/main/src/main/scala/sbt/Keys.scala#L128
- default https://github.com/sbt/sbt/blob/v1.0.2/main/src/main/scala/sbt/Defaults.scala#L2966-L2972

Reply:
thanks!

I fixed it!